### DISCUSSION TITLE
Any limit on ISR revalidate?

### DISCUSSION THREAD

**Discussion by ChristopherBanas** [vercel/next.js#42327] [Help] [answered]
I'm planning on only revalidating certain pages once every 3 days as I will only be updating my data once every 3 days. This would result in the revalidate parameter for ISR static generation being `432000`. Is there any limit for how large ISR revalidating can be?

**Comment by dubbajubba** [comment, marked as answer]
I believe the max caching time period is 31 days. https://vercel.com/docs/concepts/edge-network/caching#limits

**Reply by ChristopherBanas** [reply]
Thank you. I decided to go with ISR and call upon my endpoint when I run my data refreshes.

**Reply by dubbajubba** [reply]
Nice, that's a good idea!
